applications/maplab-node/src/maplab-node.cc

```diff
@@ -65,6 +67,31 @@ MaplabNode::MaplabNode(
   }
   CHECK(sensor_manager_);
 
+  // At the very beginning check if the main ncamera images need to be resized.
+  aslam::NCamera::Ptr mapping_ncamera =
+      vi_map::getSelectedNCamera(*sensor_manager_);
+  if (mapping_ncamera && fabs(FLAGS_image_resize_factor - 1.0) > 1e-6) {
```

Comment:
Tiny: maybe also convert the `if` of the `mapping_camera` into a check as below.

Reply:
The camera is optional here if you check later when the visual mapping is initialized